pages/Configuring/XWayland.md

```diff
@@ -3,7 +3,7 @@ compositors.
 
 # HiDPI XWayland
 
-Currently, XWayland on HiDPI screens looks pixelated/blurry, due to Xorg's
+Currently XWayland on HiDPI screens looks pixelated/blurry, due to Xorg's
```

Comment:
why did you remove the perfectly reasonable comma

Reply:
Better question, is "currently" even correct there? The correct way of using "currently" here would be 
> XWayland on HiDPI screens currently looks pixelated...

Reply:
Better yet:
> XWayland currently looks pixelated/blurry on HiDPI screens, due to Xorg's
inability to scale.